### ISSUE TITLE
A11y_Visual Studio Code Client_Problems Panel_Screen reader: Narrator/NVDA is not announcing the complete information of status bar & how to navigate.

### ISSUE THREAD

**Issue by Shivanikupatkar** [microsoft/vscode#137213]
### GitHubTags
#A11yTCS; #A11ySev2; #A11yMAS; #A11yWCAG; #DesktopApp; #Visual Studio Code Client;#BM-VisualStudioCode-Win32-Nov2021;#WCAG1.3.1; #Win32; #ScreenReader; #Element:StatusBar #NVDA; #Narrator;#Win10; #AINotInScope;

### Actual Result
Narrator/NVDA is not announcing the  complete information of the status bar.

### Expected Result
 Narrator/NVDA should announce the complete information of the status bar .

### Environment Details
Application Name: Visual Studio Code Client
version:1.62.1 (user setup)
Windows Version: Windows 10 (21H1)

### Repro Steps
1)Start Narrator/NVDA.
2)Open Visual studio code editor
2) Press ctrl+shift+M
3)Validate it's possible to navigate between items in the problems view and observe that whether screen reader is announcing the complete information of the status bar.

### User Impact
Screen reader user will face difficulty as complete information is not provided to users by screen reader.

![A11y_Visual Studio Code Client_Problems Panel_Screen reader](https://user-images.githubusercontent.com/94344363/141764896-d6d0cdea-6fdd-4cb3-b1a8-0a619ed8552f.gif)
![A11y_Visual Studio Code Client_Problems Panel_Screen reader_nvda](https://user-images.githubusercontent.com/94344363/141765066-4b2e847b-4545-44f3-8983-df9c215eab0e.gif)

**Comment by isidorn** [comment]
@Shivanikupatkar what information is actually missing?
What are you expecting the screen reader announces which it does not?

**Comment by Shivanikupatkar** [comment]
issue is that when focus is on status related things like "no problem", "Python version" .when we press tab it is just announcing the status not announcing press right arrow to navigate.

**Comment by isidorn** [comment]
@Shivanikupatkar we got explicit feedback from our screen reader community that announcing press left / right to navigate is not concise enough and just brings noise.
If we receive user that requests that we announce "press arrows to navigate" I will be open to announce this as well. For now, no change planned here.
